### PR TITLE
docs: JR-486: Updates quick start prerequisites to include links

### PR DESCRIPTION
**Pull request by bredamc**

### CODE REVIEW

**Comment by bredamc** [comment]
@jbyrne-redhat @smccarthy-ie @MelissaFlinn Can you please review the proposed updates for your respective services?

@riccardo-forina The modular-docs tests are failing for the "build" for each service -- seems the specified Ruby version is deprecated, how do we update it?

```
Use Ruby 2.6

Run actions/setup-ruby@v1
------------------------
NOTE: This action is deprecated and is no longer maintained.
Please, migrate to https://github.com/ruby/setup-ruby, which is being actively maintained.
------------------------
Error: Version 2.6 not found
```

**Review by MelissaFlinn — APPROVED** [review]
LGTM

File: docs/kafka/quarkus-kafka/quickstart.yml [hunk 1]
```diff
@@ -16,12 +16,12 @@ spec:
...
   description: !snippet README.adoc#description
   prerequisites:
-    - A running Kafka instance (see the Getting Started quick start)
+    - A running Kafka instance (see <a href="https://console.redhat.com/application-services/learning-resources?quickstart=getting-started">Getting started with OpenShift Streams for Apache Kafka</a>)
     - A command-line terminal application
     - Git
     - An IDE
     - JDK 11 or later (the latest LTS version of OpenJDK is recommended)
-    - Apache Maven 3.6.2 or later
+    - Apache Maven 3.6.2 or later 3.x
```
Comment (jbyrne-redhat) [inline docs/kafka/quarkus-kafka/quickstart.yml:24]: Not sure what the intent here is, @bredamc? Is it a mistake?

File: docs/rhoas/rhoas-service-contexts/README.adoc [hunk 2]
```diff
@@ -237,11 +237,15 @@ The Quarkus application uses a topic in a Kafka instance to produce and consume
...
+ifndef::community[]
+* You have a {org-name} account.
+endif::[]
 * You have a service context with a Kafka and {registry} instance.
 * https://github.com/git-guides/[Git^] is installed.
 * You have an IDE such as https://www.jetbrains.com/idea/download/[IntelliJ IDEA^], https://www.eclipse.org/downloads/[Eclipse^], or https://code.visualstudio.com/Download[Visual Studio Code^].
-* https://adoptopenjdk.net/[OpenJDK^] 11 or later is installed on Linux or MacOS. (The latest LTS version of OpenJDK is recommended.)
-* https://maven.apache.org/[Apache Maven^] 3.8.x or later is installed (for Quarkus 2.2.x).
+* https://adoptopenjdk.net/[OpenJDK^] 11 or later is installed. (The latest LTS version of OpenJDK is recommended.)
+* https://maven.apache.org/[Apache Maven^] 3.8.x or later 3.x is installed.
```
Comment (jbyrne-redhat) [inline docs/rhoas/rhoas-service-contexts/README.adoc:248]: Same comment as https://github.com/redhat-developer/app-services-guides/pull/598/files#r1044277775.

Unless I'm misreading this, it looks like a mistake?

**Review by jbyrne-redhat — COMMENTED** [review]
I reviewed the RHOSAK and RHOAS updates, and they look good to me, @bredamc. Just had minor questions on two lines.

File: docs/rhoas/rhoas-service-contexts/README.adoc (hunk 2 above)
Comment (bredamc) [reply docs/rhoas/rhoas-service-contexts/README.adoc:248]: Thanks, John -- the intention is to show that it is only later 3.x versions that are supported, not 4.x or later. Would it help if I inserted the word "version", and perhaps in parentheses?
 - "3.8.x or later 3.x version is installed"
 - "3.8.x (or later 3.x version) is installed"

Comment (jbyrne-redhat) [reply docs/rhoas/rhoas-service-contexts/README.adoc:248]: Ahhh, I see. I think the second reformulation (using parentheses) makes that much clearer :+1: I might state it as "**a** later 3.x version....", however.

Comment (jbyrne-redhat) [reply docs/rhoas/rhoas-service-contexts/README.adoc:248]: One remaining comment is that maybe we're overusing `x` here? Do we want to consider [semantic versioning](https://semver.org/) instead? That is:

```
* https://maven.apache.org/[Apache Maven^] 3.8.z (or a later 3.y version) is installed.
```
I don't know - perhaps I'm overthinking it. WDYT?

Comment (bredamc) [reply docs/rhoas/rhoas-service-contexts/README.adoc:248]: Actually, I see that according to IBM Style, we shouldn't use any such notation! How about the following rewrite instead?

`* https://maven.apache.org/[Apache Maven^] 3.8 (or a later Maven 3 release) is installed.`

**Comment by riccardo-forina** [comment]
> @riccardo-forina The modular-docs tests are failing for the "build" for each service -- seems the specified Ruby version is deprecated, how do we update it?

Hopefully this fixes it https://github.com/redhat-developer/app-services-guides/pull/609

File: docs/rhoas/rhoas-service-contexts/README.adoc (hunk 2 above)
Comment (jbyrne-redhat) [reply docs/rhoas/rhoas-service-contexts/README.adoc:248]: Ah, that's a good thing to know! Yes, I think that rewrite looks great.

**Comment by riccardo-forina** [comment]
If you rebase the build should start passing now 👍

**Comment by bredamc** [comment]
Woohoo, all checks passed -- many thank for the fix, @riccardo-forina !

@jbyrne-redhat Can you please review the latest updates, and approve the PR if you're happy with it?

**Review by jbyrne-redhat — APPROVED** [review]

**Comment by jbyrne-redhat** [comment]
> Woohoo, all checks passed -- many thank for the fix, @riccardo-forina !
> 
> @jbyrne-redhat Can you please review the latest updates, and approve the PR if you're happy with it?

Done, @bredamc. Thanks again for taking this on.

**Comment by bredamc** [comment]
Thanks John, I'll just wait for @smccarthy-ie to approve and then I'll merge :)

**Pull request merged**